src/main/kotlin/net/irisshaders/lilybot/utils/DatabaseHelper.kt

```diff
@@ -53,27 +41,20 @@ object DatabaseHelper {
 	 */
 	suspend fun clearConfig(inputGuildId: Snowflake) {
 		val collection = database.getCollection<ConfigData>()
-		collection.deleteOne(ConfigData:: guildId eq inputGuildId)
+		collection.deleteOne(ConfigData::guildId eq inputGuildId)
 	}
 
 	/**
 	 * Gets the number of points the provided [inputUserId] has in the provided [inputGuildId] from the database
 	 *
 	 * @param inputUserId The ID of the user to get the point value for
 	 * @param inputGuildId The ID of the guild the command was run in
-	 * @return null or the result from the database
+	 * @return The user and strikes from the Database
```

Comment:
I dislike this change. It's more useful the other way and there's not an issue with the old code.